sql/src/main/java/io/crate/protocols/postgres/types/BooleanType.java

```diff
@@ -35,7 +35,9 @@
 
     private static final int TYPE_LEN = 1;
     private static final int TYPE_MOD = -1;
-    private static final short DEFAULT_FORMAT_CODE = FormatCode.TEXT;
+
+    private final static byte[] TEXT_TRUE = new byte[] { 't' };
+    private final static byte[] TEXT_FALSE = new byte[] { 'f' };
     private static final Collection<ByteBuffer> TRUTH_VALUES = ImmutableSet.of(
```

Comment:
the `TRUTH_VALUES` is missing some of the (possible values)[https://www.postgresql.org/docs/current/static/datatype-boolean.html]